### PR TITLE
Update needle-server-gateway.js

### PR DESCRIPTION
**Pull request by manisi**
remove '/service' phrase from  address of "UpstreamPathTemplate" with api parameter  .

-   Please make sure the below checklist is followed for Pull Requests.

-   [ ] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
-   [ ] Tests are added where necessary
-   [ ] Documentation is added/updated where necessary
-   [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/main/CONTRIBUTING.md) are followed

### CODE REVIEW

File: generators/server/needle-api/needle-server-gateway.js [hunk 1]
```diff
@@ -70,7 +70,7 @@ module.exports = class extends needleBase {
                                     |    "Type": "LeastConnection"
                                     |  },
                                     |  "ReRoutesCaseSensitive": false,
-                                    |  "UpstreamPathTemplate": "/services/${microserviceName}/api/${entityName}/{everything}",
+                                    |  "UpstreamPathTemplate": "/${microserviceName}/api/${entityName}/{everything}",
```
Comment (nicolas63) [inline generators/server/needle-api/needle-server-gateway.js:73]: Are you sure removes "services" works ? I have verify in angular service the url start by services 
https://github.com/jhipster/generator-jhipster/blob/main/generators/entity-client/templates/angular/src/main/webapp/app/entities/entity.service.ts.ejs#L48

Comment (manisi) [reply generators/server/needle-api/needle-server-gateway.js:73]: @nicolas63  sure ,works.

Comment (manisi) [reply generators/server/needle-api/needle-server-gateway.js:73]: with '/services' , when Edit,Delete,or View  Item   rise this Error:
"status":404,"statusText":"Not Found","url":"http://localhost:5000/services/microservice/api/bars/1","ok":false,"name":"HttpErrorResponse","message":"Http failure response for http://localhost:5000/services/microservice/api/bars/1: 404 Not Found",

Comment (nicolas63) [reply generators/server/needle-api/needle-server-gateway.js:73]: Can you share your `entity.service` ?

**Comment by manisi** [comment]
import { Injectable } from '@angular/core';
import { HttpClient, HttpResponse } from '@angular/common/http';
import { Observable } from 'rxjs';

import { SERVER_API_URL } from 'app/app.constants';
import { createRequestOption } from 'app/shared/util/request-util';
import { IFoo } from 'app/shared/model/DarmanMicroservice/foo.model';

type EntityResponseType = HttpResponse<IFoo>;
type EntityArrayResponseType = HttpResponse<IFoo[]>;

@Injectable({ providedIn: 'root' })
export class FooService {
public resourceUrl = SERVER_API_URL + 'services/darmanmicroservice/api/foos'
;

constructor(protected http: HttpClient) {}

create(foo: IFoo): Observable<EntityResponseType> {
return this.http.post<IFoo>(this.resourceUrl, foo, { observe: 'response' });
}

update(foo: IFoo): Observable<EntityResponseType> {
return this.http.put<IFoo>(this.resourceUrl, foo, { observe: 'response' });
}

find(id: number): Observable<EntityResponseType> {
return this.http.get<IFoo>(`${this.resourceUrl}/${id}`, { observe:
'response' });
}

query(req?: any): Observable<EntityArrayResponseType> {
const options = createRequestOption(req);
return this.http.get<IFoo[]>(this.resourceUrl, { params: options, observe:
'response' });
}

delete(id: number): Observable<HttpResponse<{}>> {
return this.http.delete(`${this.resourceUrl}/${id}`, { observe: 'response'
});
}
}


‫‪nicolas63‬‏ <‪notifications@github.com‬‏> در تاریخ چهارشنبه ۱۱ نوامبر
۲۰۲۰ ساعت ۲۱:۴۵ نوشت:‬

**Comment by nicolas63** [comment]
`public resourceUrl = SERVER_API_URL + 'services/microservice/api/bars';`
The url contains services, can you verify the network and check if this url is called ?

**Comment by manisi** [comment]
Screen Shot 2020-11-11 at 23 53 00

**Comment by nicolas63** [comment]
oh ok, it's due to https://github.com/jhipster/jhipster-dotnetcore/blob/main/generators/server/templates/dotnetcore/src/Project/Configuration/MvcStartup.cs.ejs#L70 
Who map "/services" and remove this part before ocelot

**Review by nicolas63 — APPROVED** [review]
it's ok ! thanks for this bug fix

**Comment by manisi** [comment]
> it's ok ! thanks for this bug fix

You're welcome,Thanks

**Pull request merged**